src/components/CioPlpBrowse/CioPlpBrowse.tsx

```diff
@@ -0,0 +1,122 @@
+import React, { useState } from 'react';
+import { GetBrowseResultsResponse } from '@constructor-io/constructorio-client-javascript/lib/types';
+import ProductCard from '../ProductCard';
+import Filters from '../Filters';
+import FiltersIcon from '../Filters/FiltersIcon';
+import MobileModal from '../MobileModal';
+import Sort from '../Sort';
+import Pagination from '../Pagination';
+import Spinner from '../Spinner';
+import { RequestStatus } from './reducer';
+import { IncludeRenderProps } from '../../types';
+import { UsePaginationProps } from '../../hooks/usePagination';
+import { UseSortProps } from '../../hooks/useSort';
+import { UseFilterProps } from '../../hooks/useFilter';
+import useBrowseResults, { UseBrowseResultsReturn } from '../../hooks/useBrowseResults';
+
+export type CioPlpBrowseProps = {
+  initialResponse?: GetBrowseResultsResponse;
+  spinner?: React.ReactNode;
+  /**
+   * Used to set `windowSize` of `pages` array. Can also override `resultsPerPage` set at the Provider-level.
+   */
+  paginationConfigs?: Omit<UsePaginationProps, 'totalNumResults'>;
+  /**
+   * No configurations available yet.
+   */
+  sortConfigs?: Omit<UseSortProps, 'sortOptions'>;
+  /**
+   * No configurations available yet.
+   */
+  filterConfigs?: Omit<UseFilterProps, 'facets'>;
+};
+export type CioPlpBrowseWithRenderProps = IncludeRenderProps<
+  CioPlpBrowseProps,
+  UseBrowseResultsReturn
+>;
+
+export default function CioPlpGrid(props: CioPlpBrowseWithRenderProps) {
```

Comment:
I don't think I like having a separate component for what is largely the same code. Any changes we make to the structure will have to be done in both places.

I am thinking we can just have the logic to decide to use search/browse results contained in the original `CioPlpGrid` component.

If we wanted to keep the logic separate I think we can create CioPlpSearch and have CioPlpSearch/CioPlpBrowse both use the CioPlpGrid component. Likely we also want to infer whether to show search or browse results based on the url state so we will need to add logic to handle that as well.